bot/seasons/evergreen/8bitify.py

```diff
@@ -0,0 +1,54 @@
+from io import BytesIO
+
+import discord
+from PIL import Image
+from discord.ext import commands
+
+
+class EightBitify(commands.Cog):
+    """Make your avatar 8bit!"""
+
+    def __init__(self, bot):
+        self.bot = bot
+
+    @staticmethod
+    def pixelate(image: Image):
+        """Takes an image and pixelates it"""
+        return image.resize((32, 32)).resize((1024, 1024))
+
+    @staticmethod
+    def quantize(image: Image):
```

Comment:
Same as the previous one

```suggestion
    def quantize(image: Image) -> Image:
```